kleva-j/iReporter
# fix(server): fix validator functions

#### What does this PR do?
- Ensures an admin user can update a user's record status

#### Description of the task to be completed
 - create functions to authenticate a user credentials
 - create functions to update a user's record status

#### How should this be manually tested?
- Clone the repository
- Check out into the branch Feature/#162736990/update-user-status
- Install all dependencies
- run `npm start` to run the app successfully.

#### What are the relevant pivotal tracker stories?
[162736990](https://www.pivotaltracker.com/story/show/162736990)
